test_framework/terraform/aws/oracle/user-data-scripts/provision_k3s_server.sh.tpl

```diff
@@ -11,7 +11,7 @@ sudo yum install -y iscsi-initiator-utils nfs-utils nfs4-acl-tools jq rsync
 sudo systemctl -q enable iscsid
 sudo systemctl start iscsid
 
-until (curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="server --node-taint "node-role.kubernetes.io/master=true:NoExecute" --node-taint "node-role.kubernetes.io/master=true:NoSchedule" --tls-san ${k3s_server_public_ip} --write-kubeconfig-mode 644 --token ${k3s_cluster_secret}" INSTALL_K3S_VERSION="${k3s_version}" sh -); do
+until (curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="server --node-taint "node-role.kubernetes.io/master=true:NoExecute" --node-taint "node-role.kubernetes.io/master=true:NoSchedule" --tls-san ${k3s_server_public_ip} --write-kubeconfig-mode 644 --token ${k3s_cluster_secret} --selinux=true" INSTALL_K3S_VERSION="${k3s_version}" sh -); do
```

Comment:
Why for oracle, it's always enabled by default?

Reply:
Because selinux is enabled for oracle by default, and currently there is no option in our oracle terraform pipeline script to adjust the value.

But of course we can provide the option to set it to permissive/enforcing just like other selinux distro. Do we need to add this?

Reply:
I feel no need since enabling selinux is a superset of disabling.